api/doc/openapi.yaml

```diff
@@ -975,44 +966,35 @@ components:
         - $ref: '#/components/schemas/GenesisTransaction'
         - $ref: '#/components/schemas/UserTransaction'
         - $ref: '#/components/schemas/BlockMetadataTransaction'
-      discriminator:
-        propertyName: type
-        mapping:
-          pending_transaction: '#/components/schemas/PendingTransaction'
-          genesis_transaction: '#/components/schemas/GenesisTransaction'
-          user_transaction: '#/components/schemas/UserTransaction'
-          block_metadata_transaction: '#/components/schemas/BlockMetadataTransaction'
     SubmitTransactionRequest:
+      title: Submit Transaction Request
+      type: object
       allOf:
         - $ref: '#/components/schemas/UserTransactionRequest'
         - $ref: '#/components/schemas/UserTransactionSignature'
-    PendingTransactionProperties:
-      type: object
-      required:
-        - type
-        - hash
-      properties:
-        type:
-          type: string
-          example: "pending_transaction"
-        hash:
-          $ref: '#/components/schemas/HexEncodedBytes'
     PendingTransaction:
+      title: Pending Transaction
+      type: object
       allOf:
-        - $ref: '#/components/schemas/PendingTransactionProperties'
-        - $ref: '#/components/schemas/SubmitTransactionRequest'
+        - required:
```

Comment:
It is confusing to make it standalone type showing up in sidebar, so inlined `PendingTransactionProperties`.
Pulled out of `type: object`, otherwise `spotlight/elements` won't render the type correct (it will group each type under the `allOf` into group)